src/index.html

```diff
@@ -4,9 +4,206 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500&display=swap&subset=cyrillic" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo--link" href="#">
+        <img src="./images/logo.png" alt="moyo_logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav--list">
```

Comment:
```suggestion
        <ul class="nav__list">
```
block__element--modificator

`nav` `nav__list` `nav__link` `nav__link--active`